.github/workflows/runner-e2e.yml

```diff
@@ -55,6 +55,7 @@ jobs:
       run: |
         cd dist
         tar -zxvf apache-apisix-java-plugin-runner-*bin.tar.gz
+        chmod 777 /tmp/runner.sock
```

Comment:
why we need to change mod for sock file in test?

Reply:
Sorry that was a mistake and indeed as in #316 it is not needed.

I removed this line...